selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java

```diff
@@ -64,7 +66,13 @@ public void tearDown() throws Exception {
 
   @Test
   public void factoryWithDirectUrlWithSpecificBranch() throws Exception {
-    testFactoryWithSpecificBranch.authenticateAndOpen();
+    try {
+
+      testFactoryWithSpecificBranch.authenticateAndOpen();
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7555");
```

Comment:
It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.